src/Storage/Storage.Management/help/Add-AzStorageAccountManagementPolicyAction.md

```diff
@@ -0,0 +1,150 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Storage.Management.dll-Help.xml
+Module Name: Az.Storage
+online version: https://docs.microsoft.com/en-us/powershell/module/Az.storage/add-Azstorageaccountmanagementpolicyaction
+schema: 2.0.0
+---
+
+# Add-AzStorageAccountManagementPolicyAction
+
+## SYNOPSIS
+Adds an action to the input ManagementPolicy Action Group object, or creates a ManagementPolicy Action Group object with the action. The object can be used in New-AzStorageAccountManagementPolicyRule.
+
+## SYNTAX
+
+### BaseBlob (Default)
+```
+Add-AzStorageAccountManagementPolicyAction -BaseBlobAction <String> -DaysAfterModificationGreaterThan <Int32>
+ [-InputObject <PSManagementPolicyActionGroup>] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### Sanpshot
+```
+Add-AzStorageAccountManagementPolicyAction -SnapshotAction <String> -DaysAfterCreationGreaterThan <Int32>
+ [-InputObject <PSManagementPolicyActionGroup>] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Add-AzStorageAccountManagementPolicyAction** cmdlet adds an action to the input ManagementPolicy Action Group object, or creates a ManagementPolicy Action Group object with the action.
+
+## EXAMPLES
+
+### Example 1: Creates a ManagementPolicy Action Group object with 4 actions
+```
+PS C:\>$action = Add-AzStorageAccountManagementPolicyAction -BaseBlobAction Delete -daysAfterModificationGreaterThan 100
+PS C:\>$action = Add-AzStorageAccountManagementPolicyAction -BaseBlobAction TierToArchive -daysAfterModificationGreaterThan 50  -InputObject $action
+PS C:\>$action = Add-AzStorageAccountManagementPolicyAction -BaseBlobAction TierToCool -daysAfterModificationGreaterThan 30 -InputObject $action
+PS C:\>$action = Add-AzStorageAccountManagementPolicyAction -SnapshotAction Delete -daysAfterCreationGreaterThan 100 -InputObject $action
```

Comment:
Please add example output for all cmdlets in all help files. Please also show not only the cmdlet itself, but also how it is used in conjunction with the other cmdlets in this scenario (though you only need to show example output for the cmdlet in question).

Reply:
Fixed. 
All cmdlets has output sample in help.
And all related cmdlets are in help of the cmdlets